src/js/view/progressbar.js

```diff
@@ -28,6 +28,14 @@ export default function ProgressBar(elements) {
                 durationElement.innerText = '--:--';
                 sliderElem.style.width = '0%';
                 mediaDuration = 0;
+            } else if (utils.streamType(duration) === 'LIVE') {
+                let durationChanged = mediaDuration != duration;
+                if (durationChanged) {
+                    mediaDuration = duration;
+                    elapsedElem.innerText = 'Live';
+                    durationElement.innerText = '';
+                    sliderElem.style.width = '100%';
```

Comment:
@RikHeijdens @hghazzi I didn't see any design guideline specific to live stream playback. My recommendation is to display `Live broadcast` without the progress bar as we do in the player.

<img width="485" alt="screen shot 2017-03-27 at 2 52 56 pm" src="https://cloud.githubusercontent.com/assets/4603952/24372823/3f2bbed8-12fd-11e7-8ffe-2761c2abcd2d.png">